modules/rhcos-add-extensions.adoc

```diff
@@ -8,10 +8,12 @@
 = Adding extensions to {op-system}
 {op-system} is a minimal container-oriented RHEL operating system, designed to provide a common set of capabilities to {product-title} clusters across all platforms. While adding software packages to {op-system} systems is generally discouraged, the MCO provides an `extensions` feature you can use to add a minimal set of features to {op-system} nodes.
 
-Currently, the following extension is available:
+Currently, the following extensions are available:
 
 * **usbguard**: Adding the `usbguard` extension protects {op-system} systems from attacks from intrusive USB devices. See link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html-single/security_hardening/index#usbguard_protecting-systems-against-intrusive-usb-devices[USBGuard] for details.
 
+* **kerberos**: Adding the `kerberos` extention provides the necessary tools for securely accessing data in your existing Kerberos-controlled file shares from within OpenShift. See link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/deploying_different_types_of_servers/securing-nfs_deploying-different-types-of-servers#configuring-an-nfs-server-and-client-to-use-kerberos_securing-nfs[_Configuring an NFS server and client to use Kerberos_] for information about how to set up a Kerberized NFS server.
```

Comment:
```suggestion
* **kerberos**: Adding the `kerberos` extension provides the necessary tools for securely accessing data in your existing Kerberos-controlled file shares from within OpenShift. See link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/deploying_different_types_of_servers/securing-nfs_deploying-different-types-of-servers#configuring-an-nfs-server-and-client-to-use-kerberos_securing-nfs[_Configuring an NFS server and client to use Kerberos_] for information about how to set up a Kerberized NFS server.
```

Reply:
+1 to s/OpenShift/{product-title}, also pretty sure that link should not be italicized (but also I am perpetually unsure about citation formatting).

Reply:
Oh yeah, forgot about the attribute. Good catch, thanks!

The other link isn't italicized so un-italicizing it will look better anyway 😄